src/mlpack/methods/ann/not_adapted/brnn_impl.hpp

```diff
@@ -384,12 +384,8 @@ EvaluateWithGradient(const arma::mat& /* parameters */,
 
   if (backwardGradient.is_empty())
   {
-    backwardGradient = arma::zeros<arma::mat>(
-        parameter.n_rows/ 2,
-        parameter.n_cols);
-    forwardGradient = arma::zeros<arma::mat>(
-        parameter.n_rows/ 2,
-        parameter.n_cols);
+    backwardGradient = zeros<arma::mat>(parameter.n_rows/ 2, parameter.n_cols);
+    forwardGradient = zeros<arma::mat>(parameter.n_rows/ 2, parameter.n_cols);
```

Comment:
```suggestion
    backwardGradient = zeros<arma::mat>(parameter.n_rows / 2, parameter.n_cols);
    forwardGradient = zeros<arma::mat>(parameter.n_rows / 2, parameter.n_cols);
```
(I did check that it still fits in 80 characters even with the space :))